src/app/Utils/UpdateValidation.ts

```diff
@@ -0,0 +1,7 @@
+import * as yup from 'yup';
+export const schemaUpdateTranslation = yup.object({
+    key: yup.string().required('Key is required'),
+    text: yup.string().required('Text is required'),
+    UpdateLanguages: yup.array().min(1).required('At least one language is required'),
+  }).required();
```

Comment:
why you make all the object is required as you make all of its field to be required??